pkg/cmd/datastore/datastore.go

```diff
@@ -104,6 +104,7 @@ func RegisterDatastoreFlags(cmd *cobra.Command, opts *Config) {
 	cmd.Flags().DurationVar(&opts.RequestHedgingInitialSlowValue, "datastore-request-hedging-initial-slow-value", 10*time.Millisecond, "initial value to use for slow datastore requests, before statistics have been collected")
 	cmd.Flags().Uint64Var(&opts.RequestHedgingMaxRequests, "datastore-request-hedging-max-requests", 1_000_000, "maximum number of historical requests to consider")
 	cmd.Flags().Float64Var(&opts.RequestHedgingQuantile, "datastore-request-hedging-quantile", 0.95, "quantile of historical datastore request time over which a request will be considered slow")
+	cmd.Flags().BoolVar(&opts.EnableDatastoreMetrics, "datastore-prometheus-metrics", true, "set to false to disabled prometheus metrics from the datastore")
```

Comment:
this does not seem to align with the default in the individual stores? e.g. https://github.com/authzed/spicedb/pull/561/files#diff-1f9d83984c96749a94659b71f4b5b8366b0b98e11e71923100983d110cd7b3a3R50

Reply:
Correct. It should be enabled by default from the CLI, but disabled by default if you just instantiate the datastore directly

Reply:
got it, it's intentional 👍🏻